packages/clerk-js/src/core/clerk.ts

```diff
@@ -980,8 +980,11 @@ export default class Clerk implements ClerkInterface {
   };
 
   #loadInStandardBrowser = async (): Promise<boolean> => {
+    this.#authService = new AuthenticationService(this);
     if (this.isSatellite && this.#options.shouldSyncLink) {
       if (!this.#hasSynced()) {
+        // set session cookie to avoid throwing an extra interstitial
+        this.#authService?.setAuthCookiesFromSession(this.session);
```

Comment:
❓ I'm certainly missing context, but do we have a `this.session` at this point? The `/client` request happens after that if I'm not mistaken.